tests/v2_validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -2903,3 +2904,116 @@ def create_service_ingress(ingresses, services, port, namespace,
         lbips.append(lb_ip)
 
     return(podnames, lbips)
+
+
+@pytest.fixture(scope='session')
+def rancher_cli_container(admin_client, client, request):
+
+    if rancher_cli_con["container"] is not None:
+        return
+    setting = admin_client.by_id_setting(
+        "default.cattle.rancher.cli.linux.url")
+
+    rancher_cli_url = setting.value
+    print "Value is"
+    print setting.value
+    cmd1 = \
+        "wget " + rancher_cli_url
+    rancher_cli_file = rancher_cli_url.split("/")[-1]
+
+    cmd2 = "tar xvf " + rancher_cli_file
+    print cmd2
+
+    hosts = client.list_host(kind='docker', removed_null=True, state="active")
+    assert len(hosts) > 0
+    host = hosts[0]
+    port = rancher_cli_con["port"]
+    c = client.create_container(name="rancher-cli-client",
+                                networkMode=MANAGED_NETWORK,
+                                imageUuid="docker:sangeetha/testclient",
+                                ports=[port+":22/tcp"],
+                                requestedHostId=host.id
+                                )
+    c = client.wait_success(c, SERVICE_WAIT_TIMEOUT)
+    assert c.state == "running"
+    time.sleep(5)
+
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(host.ipAddresses()[0].address, username="root",
+                password="root", port=int(port))
+    cmd = cmd1 + ";" + cmd2
+    print cmd
+    stdin, stdout, stderr = ssh.exec_command(cmd)
+    response = stdout.readlines()
+    print "Hello rancher cli"
+    print response
+    found = False
+    for resp in response:
+        if "/rancher" in resp:
+            found = True
+    assert found
+    rancher_cli_con["container"] = c
+    rancher_cli_con["host"] = host
+
+    def remove_rancher_cli_container():
+        delete_all(client, [rancher_cli_con["container"]])
+
+    request.addfinalizer(remove_rancher_cli_container)
+
+
+def execute_rancher_cli(client, env_name, command,
+                        expected_resp, docker_compose, rancher_compose,
+                        timeout=SERVICE_WAIT_TIMEOUT):
```

Comment:
expected_Rep should be removed since it is not used
Have default values as None for docker_compose and rancher_compose
Rename env_name as stack_name